snap/snapcraft.yaml

```diff
@@ -67,16 +67,20 @@ parts:
 
   node:
     plugin: x-nodejs
-    node-packages: [postcss-cli]
+    node-packages: [postcss-cli, @babel/cli]
     filesets:
       node:
         - bin/node
       postcss:
         - bin/postcss
         - lib/node_modules/postcss-cli/*
+      babel:
+        - bin/babel.js
```

Comment:
I'm not sure which `bin` folder this is and how it gets populated from node, but npm/yarn would install the babel binary as just `babel`  without `.js`.